14-truck_backer-upper.ipynb

```diff
@@ -364,12 +364,12 @@
     "\n",
     "for i in tqdm(torch.randperm(len(train_inputs))):\n",
     "    ϕ_state = train_inputs[i]\n",
+    "    optimiser_e.zero_grad()\n",
```

Comment:
Why?

Reply:
What do you mean why?
Are you asking why did I put this here?

Reply:
Yes, why did you move it up?

Reply:
Probably I did it without knowing what I was doing.

Reply:
It does not change anything. But I like to have it just before `.backward()`.